config/main.py

```diff
@@ -19,6 +19,7 @@
 from utilities_common.intf_filter import parse_interface_in_filter
 from utilities_common.util_base import UtilHelper
 from portconfig import get_child_ports, get_port_config_file_name
+from utilities_common.common import *
```

Comment:
>* [](start = 36, length = 1)

Do not import wildcard in prod code.

Reply:
LGTM complained too. Fixed per LGTM suggestion.
"`NEWImport pollutes the enclosing namespace, as the imported module utilities_common.common does not define '__all__'.`"

```
__all__ = ['log_info', 'log_err', 'log_warning', 'log_debug',
        'do_exit', 'get_configdb_data', 'run_command',
        'AbbreviationGroup']

```